soil-service/app/services/fields/humus_balance_service.rb

```diff
@@ -0,0 +1,20 @@
+module Fields
+  class HumusBalanceService
+    MULTIPLY_EFFECT = 1.3
+
+    def initialize(crops:)
+      @crops = crops
+    end
+
+    def call
+      humus_balance = crops.each_with_index.reduce(0) do |total_humus, (humus_delta, index)|
+        effect = index.zero? ? 1 : MULTIPLY_EFFECT
+        (humus_delta + total_humus ) * effect
```

Comment:
I think your logic is bit wrong here. MULTIPLY_EFFECT should be used only for same successive crops.
Meaning if there is "Red Clover" two successive years then second year its humus delta multiplied by 1.3
At the moment you use this effect for each successive crop (if there is multiple crops)